tiledb/sm/consolidator/array_meta_consolidator.cc

```diff
@@ -80,6 +80,12 @@ Status ArrayMetaConsolidator::consolidate(
       encryption_key,
       key_length));
 
+  // Check if there's actually more than 1 file to consolidate
+  auto& metadata_r = array_for_reads.metadata();
+  if (metadata_r.loaded_metadata_uris().size() <= 1) {
```

Comment:
I had thought about using `array_for_reads.array_directory().array_meta_uris()` but [it's the same thing](https://github.com/TileDB-Inc/TileDB/blob/ef07fe983b925f2e2d8a3f2fc3965ad14063d139/tiledb/sm/array/array.cc#L1829-L1853).